Dockerfile

```diff
@@ -1,16 +1,19 @@
-FROM centos:7
+FROM docker-registry.eyeosbcn.com/alpine6-node-base
 
+COPY alpine-*.list /var/service/
 COPY serf.py /bin/
 
 ENV SERF_ADVERTISE_IP 172.17.42.1
 ENV SERF_BIND_ADDRESS ""
 
-RUN cd /tmp &&\
-    curl -L https://releases.hashicorp.com/serf/0.6.4/serf_0.6.4_linux_amd64.zip -o serf.zip &&\
-    yum install -y unzip &&\
-    unzip serf.zip &&\
-    rm serf.zip &&\
-    cp serf /usr/bin/serf &&\
+RUN cd /tmp && \
+    apk update && \
+    /scripts-base/buildDependencies.sh --production --install && \
+    curl -L https://releases.hashicorp.com/serf/0.6.4/serf_0.6.4_linux_amd64.zip -o serf.zip && \
```

Comment:
@forhau Serf is already there in the alpine6-node-base container. You do not need to install it again.